lib/watir/elements/element.rb

```diff
@@ -1,3 +1,5 @@
+# encoding: utf-8
```

Comment:
This PR is based on code I started writing over a year ago, so likely a bad merge choice, I'll remove it.